tool/site/client-react/src/pages/changeAuthPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useContext, useState} from "react"
+import {Button, Label} from "nhsuk-react-components"
+import {AppContext} from "../index"
+import {axiosInstance} from "../requests/axiosInstance"
+import ButtonList from "../components/buttonList"
+import {redirect} from "../browser/navigation"
+
+const ChangeAuthPage: React.FC = () => {
+  const {baseUrl} = useContext(AppContext)
+
+  const [pressedUserButton, setPressedUserButton] = useState(false)
+
+  return <>
+    <Label isPageHeading>Change auth</Label>
+    {!pressedUserButton ? <>
+      <Label>Select access level:</Label>
+      <ButtonList>
+        <Button type="submit" onClick={() => setPressedUserButton(true)}>User</Button>
```

Comment:
maybe a rename to help context here? setPressedUserButton -> attendedAccessSelected

Reply:
yea seen this pattern elsewhere, will pick up